### PR TITLE
HDDS-7356. Update SCM-HA.zh.md to match the English version

### PR DESCRIPTION
**Pull request by kaijchen**
## What changes were proposed in this pull request?

Update SCM-HA.zh.md, mainly on SCM-HA security.
Followup #3846.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-7356

## How was this patch tested?

No need.

### CODE REVIEW

**Review by szetszwo — COMMENTED** [review]
@kaijchen , the change looks good.  Just two minor comments inlined.

File: hadoop-hdds/docs/content/feature/SCM-HA.zh.md [hunk 1]
```diff
@@ -111,6 +113,67 @@ bin/ozone scm --bootstrap
...
+<config>hdds.grpc.tls.enabled</config>
+<value>true</value>
+</property>
+```
+
+在正常的 SCM HA 配置的基础上，需要添加上述配置。
+
+### 原始 SCM
+
+原始 SCM 由配置区区臭氧.scm.primordial.node.id 确定。
```
[the remark this answers is not in the document]
Comment (kaijchen) [reply hadoop-hdds/docs/content/feature/SCM-HA.zh.md:150]: Sorry I used Google translate to do the first pass, Ozone was translated to `臭氧` by mistake.

File: hadoop-hdds/docs/content/feature/SCM-HA.zh.md [hunk 2]
```diff
@@ -111,6 +113,67 @@ bin/ozone scm --bootstrap
...
+原始 SCM 使用自签名证书启动根 CA，并用于颁发签名证书到它自己和其他
+引导的 SCM。 只有原始 SCM 可以为其他 SCM 颁发签名证书。
+因此，原始 SCM 在 SCM HA 集群中具有特殊的作用，因为它是唯一可以向 SCM 颁发证书的 SCM。
+
+原始 SCM 担任根 CA 角色，它使用子 CA 证书签署所有 SCM 实例。
+SCM 使用子 CA 证书来签署 OM/Datanodes 的证书。
+
+引导 SCM 时会从原始 SCM 获取签名证书并启动子 CA。
+
+SCM 上的子 CA 用于为集群中的 OM/DN 颁发签名证书。 只有原始 SCM 向 OM/DN 颁发证书。
```
[the remark this answers is not in the document]
Comment (kaijchen) [reply hadoop-hdds/docs/content/feature/SCM-HA.zh.md:130]: The original text is `Primordial`, will `leader SCM` be ambiguous to Raft leader？

[the remark this answers is not in the document]
Comment (kaijchen) [reply hadoop-hdds/docs/content/feature/SCM-HA.zh.md:130]: To better reflect the difference of `root CA` vs `sub CA`,
how about using `根 SCM` (`root SCM`) and `子 SCM` (`sub SCM`)?

The English text used `Primordial SCM` and `Bootstrap SCM` though.

[the remark this answers is not in the document]
Comment (szetszwo) [reply hadoop-hdds/docs/content/feature/SCM-HA.zh.md:130]: The English use "leader SCM" for this sentence.
https://github.com/apache/ozone/blob/e45f9b8333a0ccd605bdc3d18aa36282d4ba5859/hadoop-hdds/docs/content/feature/SCM-HA.md?plain=1#L131

[the remark this answers is not in the document]
Comment (kaijchen) [reply hadoop-hdds/docs/content/feature/SCM-HA.zh.md]: Oops, thanks for pointing out.

**Review by szetszwo — APPROVED** [review]
+1 the change looks good.

**Comment by kaijchen** [comment]
Thanks @szetszwo for the review.

**Pull request merged**